etc/scripts/install-on-minishift.sh

```diff
@@ -15,6 +15,7 @@ OPENSHIFT_VERSION=${OPENSHIFT_VERSION:-v3.11.0}
 MEMORY=${MEMORY:-10GB}
 CPUS=${CPUS:-4}
 DISK_SIZE=${DISK_SIZE:-50g}
+VM_DRIVER=${VM_DRIVER:-$(minishift config get vm-driver 2>/dev/null || echo "virtualbox")}
```

Comment:
Don't think this works with latest minishift as at least my minishift returns 0 even when no vm-driver is set. Also output goes to stdout, too so I'd suggest a `2&1 >/dev/null` here,

Reply:
@rhuss Ok with your suggestions.